md-dao/src/main/java/com/wipro/ats/bdre/md/dao/ProcessDAO.java

```diff
@@ -243,14 +241,15 @@ public void testDelete(Integer id) {
         return processSubProcessList;
     }
 
-    public List<Process> selectProcessList(String processCode) {
+    public List<Process> selectProcessList(String processCode,String username) {
         Session session = sessionFactory.openSession();
         session.beginTransaction();
         List<Process> processSubProcessList = new ArrayList<Process>();
         try {
-             Process parentProcess = (Process) session.createCriteria(Process.class).add(Restrictions.eq(PROCESSCODE,processCode)).uniqueResult();
-            Criteria checkProcessSubProcessList = session.createCriteria(Process.class).add(Restrictions.or(Restrictions.eq(PROCESSCODE, processCode), Restrictions.eq(PROCESS, parentProcess)));
+             Process parentProcess = (Process) session.createCriteria(Process.class).add(Restrictions.eq(PROCESSCODE,processCode)).add(Restrictions.eq("users.username",username)).uniqueResult();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "users.username" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)